packages/cli/src/reunite/commands/push.ts

```diff
@@ -54,6 +54,12 @@ export async function handlePush({
     );
   }
 
+  if (!mountPath || mountPath === '/') {
```

Comment:
Could you check if it could be checked inside the `yargs` layer? Looks like it's more appropriate place for that.